docs/cluster-configuration.md

```diff
@@ -126,6 +126,12 @@ CPU HW perspective.
 
 Additional information: [LibvirtXMLCPUModel](https://wiki.openstack.org/wiki/LibvirtXMLCPUModel)
 
+### SRIOVLiveMigration Feature Gate
+Set the `SRIOVLiveMigration` feature gate in order to allow migrating a virtual machine with SRIOV interfaces.
```

Comment:
The name of the FG is with the wrong casing. Should be `sriovLiveMigration`

Reply:
Also, please add it to the example below.

Reply:
Done